Description: Allow greater versions for both pep8, flake8 and pyflakes
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2014-03-22

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,7 +1,7 @@
 pbr>=0.6,!=0.7,<1.0
 
-pep8==1.5.6
-pyflakes==0.8.1
-flake8==2.1.0
+pep8>=1.5.6
+pyflakes>=0.8.1
+flake8>=2.1.0
 
 six>=1.6.0
